src/compiler/commandLineParser.ts

```diff
@@ -201,6 +201,14 @@ namespace ts {
             description: Diagnostics.Enable_incremental_compilation,
             transpileOptionValue: undefined
         },
+        {
+            name: "assumeChangesOnlyAffectDirectDependencies",
+            type: "boolean",
+            affectsSemanticDiagnostics: true,
+            affectsEmit: true,
+            category: Diagnostics.Advanced_Options,
+            description: Diagnostics.Have_recompiles_in_incremental_and_watch_assume_that_changes_within_a_file_will_only_affect_files_directly_depending_on_it

```

Comment:
I suspect this is probably a very ignorant question, but does this apply only within projects or across projects as well?  Clearly we can't get everything into the name, but it might be nice to provide more details in the description (or in a a comment, if we'd prefer to keep the switch mysterious and little-used).

Reply:
Builder is not used across projects but within single program.